src/stamp/preprocessing/__init__.py

```diff
@@ -298,7 +298,8 @@ def extract_(
                 h5_fp.attrs["stamp_version"] = stamp.__version__
                 h5_fp.attrs["extractor"] = extractor_id
                 h5_fp.attrs["unit"] = "um"
-                h5_fp.attrs["tile_size"] = tile_size_um
+                h5_fp.attrs["tile_size_um"] = tile_size_um
```

Comment:
Please add a comment `# changed in v2.1.0` in case somebody needs to debug this later